apps/dapp-console-api/migrations/0012_conscious_khan.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "challenges" DROP COLUMN IF EXISTS "challenge";--> statement-breakpoint
+ALTER TABLE "challenges" DROP COLUMN IF EXISTS "reponse";
```

Comment:
dropping the challenge column. I think it is best if we generate challenges on request instead of storing in db. this gives us the ability to change challenges later if we need to. 

dropping the response column because there is no need to store signatures for this feature.

Reply:
i think this is fine, something to consider is if we ever changed this while there were pending challenges out it could cause them to start failing if we're not keeping track of them or what version of a challenge is being signed. for an initial version of this though i think this is fine until we otherwise run into a need to keep it in the db